src/app.js

```diff
@@ -27,10 +27,14 @@ import './noise';
 import AudioIn from './audioin';
 p5.AudioIn = AudioIn;
 
-import Effect from './effect' ;
+import Effect from './effect';
 p5.Effect = Effect;
 
-import './filter';
+import { Filter, LowPass, HighPass, BandPass } from './filter';
```

Comment:
I think Filter should be the default export, just to stay consistent with other modules where the filename is the name of the default export

Reply:
done the required changes ! have a look!